$MirOS: ports/www/dillo/v2/patches/patch-src_cookies_c,v 1.2 2009/08/23 15:27:58 bsiegert Exp $
--- src/cookies.c.orig	Mon Jan 18 18:00:19 2010
+++ src/cookies.c	Sun Feb 21 18:17:00 2010
@@ -238,7 +238,7 @@ static int Cookie_control_init(void)
    bool_t enabled = FALSE;
 
    /* Get a file pointer */
-   filename = dStrconcat(dGethomedir(), "/.dillo/cookiesrc", NULL);
+   filename = dStrconcat(dGethomedir(), "/.etc/dillo/cookiesrc", NULL);
    stream = Cookies_fopen(filename, "DEFAULT DENY\n");
    dFree(filename);
 
